blocksync/blocksync.go

```diff
@@ -85,12 +89,18 @@ func (bs *blockSyncer) P2P() network.Overlay {
 // Start starts a block syncer
 func (bs *blockSyncer) Start(ctx context.Context) error {
 	logger.Debug().Msg("Starting block syncer")
+	if err := bs.chaser.Start(ctx); err != nil {
+		return err
+	}
 	return bs.worker.Start(ctx)
 }
 
 // Stop stops a block syncer
 func (bs *blockSyncer) Stop(ctx context.Context) error {
 	logger.Debug().Msg("Stopping block syncer")
+	if err := bs.chaser.Stop(ctx); err != nil {
```

Comment:
do we have any rule of when to the order of start/stop modules? As bs exends lifecycle, we may add it to bs via Add